healthcare/v1beta1/src/main/java/snippets/healthcare/dicom/DicomWebRetrieveStudy.java

```diff
@@ -90,11 +93,14 @@ private static CloudHealthcare createClient() throws IOException {
         GoogleCredential.getApplicationDefault(HTTP_TRANSPORT, JSON_FACTORY)
             .createScoped(Collections.singleton(CloudHealthcareScopes.CLOUD_PLATFORM));
 
+    HttpHeaders headers = new HttpHeaders();
+    headers.set("X-GFE-SSL", "yes");
```

Comment:
Probably should explain this.

Reply:
Added an explanation.

Reply:
where?

Reply:
found it.

Reply:
I was speaking about the X-GFE-SSL header as well.

Reply:
That's in all of the samples that dzlier wrote. Should I add it to every sample as well?

Reply:
Is a customer likely to see this w/o seeing dziller's code?  If so, you should explain it.  Frankly, I hadn't seen it before.

We can't explain things enough.  The standard I use is, if a user familiar with coding, the language, comes to this code, will they understand it, if not, we should comment.

Reply:
I spoke to dzlier and he said that it's actually not necessary. I'll need to go through all of the other Healthcare samples and remove it wherever it's OK to, but for now I've removed it from this sample.

Reply:
Cool